cfme/utils/ext_auth.py

```diff
@@ -86,7 +86,7 @@ def setup_external_auth_openldap(**data):
         'password': credentials['host_default']['password'],
         'hostname': data['ipaddress'],
     }
-    current_appliance = get_or_create_current_appliance()
+    current_appliance = appliance.current_appliance
```

Comment:
switching `get_or_create_current_appliance` for a local proxy that wraps around it makes things actually worse

Reply:
heh, gotcha. I can switch to all of them using get_or_create_current_appliance

Reply:
what would be even better would be turning them into actual arguments, but when i last tried that the follow-up trail tried to strangle me

Reply:
Agreed - Was looking to keep the scope small here to resolve some of the errors. I'm going to be going through the auth tests, and will audit the use of these utils, I'm pretty sure that I'll be able to convert these methods to fixtures.